projects/physics/public/math.js

```diff
@@ -0,0 +1,92 @@
+//vector manipulation, stored in radians
+const vector = (degrees, magnitude) => {
+  return ({ angle: degrees * Math.PI / 180, magnitude });
+};
+
+const add2Vectors = (v1, v2) => {
+  const x1 = Math.cos(v1.angle) * v1.magnitude;
+  const x2 = Math.cos(v2.angle) * v2.magnitude;
+  const y1 = Math.sin(v1.angle) * v1.magnitude;
+  const y2 = Math.sin(v2.angle) * v2.magnitude;
+  const angle = Math.atan2(y1 + y2, x1 + x2);
+  const mag = Math.hypot((x1 - x2), (y1 - y2));
+  return ({ angle, magnitude: mag });
```

Comment:
Mostly just to show you a Github feature, here's a suggestion which I created by clicking this icon:

![2023-01-08 at 11 12 AM](https://user-images.githubusercontent.com/250053/211214437-86f09589-4fa5-49fe-bfc6-a5dc79df4f2d.png)

```suggestion
  const magnitude = Math.hypot((x1 - x2), (y1 - y2));
  return { angle, magnitude };
```

You should be able to accept this charge directly into the PR by clicking a button.